Mathlib/Order/Rel/GaloisConnection.lean

```diff
@@ -58,7 +58,8 @@ theorem gc_leftDual_rightDual : GaloisConnection (toDual ∘ R.leftDual) (R.righ
 /-- `leftFixedPoints` is the set of elements `J : Set α` satisfying `rightDual (leftDual J) = J`. -/
 def leftFixedPoints := {J : Set α | R.rightDual (R.leftDual J) = J}
 
-/-- `rightFixedPoints` is the set of elements `I : Set β` satisfying `leftDual (rightDual I) = I`. -/
+/-- `rightFixedPoints` is the set of elements `I : Set β` satisfying `leftDual (rightDual I) = I`.
```

Comment:
Personally, I'd like `/--` on its own line but I don't know the community's sentiment.

Reply:
My understanding of it is "on a separate line is fine for single sentences/up to two lines; separate paragraphs implies delimiters on separate lines". Basically, close Lean core style (but allowing
```
/-- Two lines of 
doc comment -/
```
This one seems to be in accordance with it.

Reply:
I tried to maintain the symmetry with the one above, that is why I went for a minimalistic change.